TrackletAlgorithm/MatchProcessor.h

```diff
@@ -803,51 +820,104 @@ void MatchProcessor(BXType bx,
 
       unsigned int ivmPlus = iphi;
 	
-      ap_int<2> shift = 0;
+      //ap_int<2> shift = 0;
       
+      //If the projection goes to the first (ivmMinus) phiProjBin is zero but
+      //if the projection goes to the second bin ivmMinus+1 phiProjBin is one
+      ap_uint<1> phiProjBin = 0; 
```

Comment:
@jasonfan393 I guess we can elimimate a magic number by taking the type from `ProjectionRouterBuffer`:
```suggestion
      ProjectionRouterBuffer<BARREL, APTYPE>::PHIPROJBIN phiProjBin = 0;
```
Maybe other magic numbers could be eliminated in a similar way…